experimental/client-app/README.md

```diff
@@ -18,12 +18,61 @@ Within your terminal, open one tab to run the proxy and another to run the app.
 ```bash
 npm install
 npm build 
-npm run dev
+npm run start
 ```
 
-- `server` runs the proxy server to target api on port `3000`
-- `start` runs your app for development on port `8000`, reloading on file changes
-- `dev` uses npm-run-all to run the above `server` and `start` scripts in parallel (Perfect for cloudshell development.)
+To run locally:
+
+- `npm start` runs your app for development on port `8000`, reloading on file changes
+
+## Deploying Lit container
+
+```
+// Requires the following environment
+// variables to be set Cloud Run container
+REDIRECT_URI - uri pointing to auth api callback (i.e */auth/google)
+API_URL - url that points to server api to obtain cause/campaign information
```

Comment:
Can we provide example values for each of these?

Also, re-organizing this into a Markdown table might make it easier to read. 😃